### PR TITLE
Milestone 8: Pop-up Window

### PR DESCRIPTION
**Pull request by Xghayor**
In this milestone, I implemented a pop-up window feature for my portfolio website. The pop-up window allows users to view more details about a specific project when they click on a project button.

Pop-up Window Functionality:

Created a project data array to store information about different projects.
Selected the necessary elements from the DOM, such as the project button, modal, image, title, description, and technology list.
Added an event listener to the project button, triggering the display of the modal when clicked.
Updated the modal content with the relevant project information, including the featured image, title, description, and list of technologies.
Displayed the modal by changing its style to "block."

### CODE REVIEW

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: - [ ] The width of your popup isn't consistent with the figma desgin template as shown below. Please kindly see the red arrows showing the excess space to the left and right and make the appropriate corrections. :+1: 
- figma design
gb
- your design
gbs

Comment (Efiamotu-1) [inline index.html:1]: - [ ] Your design for the popup doesn't have a space at the bottom which is not the same as the provided figma design please kindly fix this before requesting a new review. :+1: 
**your design**
gb

File: index.html [hunk 1]
```diff
@@ -61,8 +61,6 @@ <h2 class="prof">I’m a Software Developer</h2>
     <h3 class="works-h3">My Recent Work</h3>
     <hr class="works-hr">
   </div>
-  
-
 
 <div class="box-container">
```
Comment (Efiamotu-1) [inline index.html:65]: - Your works section should be generated dynamically using JavaScript, which means you would not have any hard coded html. currently you still have html code which shouldn't be the case i kindly suggest you work on creating your works section dynamically :+1: 
goof

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Xghayor 
 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Shedrack-Sunday) [inline index.html:65]: - [ ] As suggested by the previous reviwer, the main `index.html` content should be dynamically generated using javascript based on project requirements. I can see headings and skills section still static here.
- You can use [This link](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and[This](https://github.com/nicupop729/Portfolio/pull/11/commits/aabed37cc2208f52d0bd081a4cae8cdc83c49636) to see sample implementaion

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @Xghayor      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Nice work with the setup✔️ 
- Nice effort on the suggested changes previously 👍   
  
# Required Changes  🌵  

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Xghayor) [reply index.html:65]: @Shedrack-Sunday 
i have created the whole modal and grids dynamically using Javascript, can you please show me the modal section in my html?

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @Xghayor , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Design looks very similar to figma design. :+1: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**